.agents/skills/release-notes/scripts/verify.py

```
@@ -0,0 +1,100 @@
+        if not url.startswith("https://"):
+            errors.append(f"NOT HTTPS: {url}")
+            continue
+
+        # Validate host is safe before making any requests
+        is_safe, err_msg = is_safe_host(url)
+        if not is_safe:
+            errors.append(f"UNSAFE HOST ({err_msg}): {url}")
+            continue
+
+        try:
+            req = urllib.request.Request(url, method="HEAD", headers={"User-Agent": "url-verify/1.0"})
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)